build.sbt

```diff
@@ -82,9 +82,11 @@ lazy val orchardWS = (project in file("orchard-ws")).
       scalaPlusPlay
     ),
     dependencyOverrides ++= Seq(
-      // fix https://nvd.nist.gov/vuln/detail/CVE-2020-36518
-      "com.fasterxml.jackson.core" % "jackson-databind" % "2.13.4",
-      "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.13.4"
+      // fix https://nvd.nist.gov/vuln/detail/CVE-2022-42003
+      "com.fasterxml.jackson.core" % "jackson-databind" % "2.13.4.1",
+      "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.13.4",
+      // fix https://nvd.nist.gov/vuln/detail/CVE-2022-3171
+      "com.google.protobuf" % "protobuf-java" % "3.21.7"
```

Comment:
@damianxu88 I cannot find protobuf-java in the dependencies

Reply:
@realstraw just double checked, this is a flyway dependency, let me remove